examples/bilingual_document.ipynb

```diff
@@ -0,0 +1,305 @@
', 'But he retracted it immediately, for the contact felt like a cold shower all over him.', 'He slid back again into his earlier position.', '\"This getting up early,\" he thought, \"makes a man quite idiotic.', 'A man must have his sleep.', 'Other traveling salesmen live like harem women.', 'For instance, when I come back to the inn during the course of the morning to write up the necessary orders, these gentlemen are just sitting down to breakfast.', \"If I were to try that with my boss, I'd be thrown out on the spot.\", \"Still, who knows whether that mightn't be really good for me.\", \"If I didn't hold back for my parents' sake, I would've quit ages ago.\", \"I would've gone to the boss and told him just what I think from the bottom of my heart.\", \"He would've fallen right off his desk!\", 'How weird it is to sit up at the desk and talk down to the employee from way up there.', 'The boss has trouble hearing, so the employee has to step up quite close to him.', \"Anyway, I haven't completely given up that hope yet.\", \"Once I've got together the money to pay off the parents' debt to him--that should take another five or six years--I'll do it for sure.\", \"Then I'll make the big break.\", 'In any case, right now I have to get up.', 'My train leaves at five o\\'clock.\"', 'And he looked over at the alarm clock ticking away by the chest of drawers.', '\"Good God,\" he thought.', 'It was half past six, and the hands were going quietly on.', 'It was past the half hour, already nearly quarter to.', 'Could the alarm have failed to ring?', \"One saw from the bed that it was properly set for four o'clock.\", 'Certainly it had rung.', 'Yes, but was it possible to sleep through this noise that made the furniture shake?', \"Now, it's true he'd not slept quietly, but evidently he'd slept all the more deeply.\", 'Still, what should he do now?', \"The next train left at seven o'clock.\", 'To catch that one, he would have to go in a mad rush.', \"The sample collection wasn't packed up yet, and he really didn't feel particularly fresh and active.\", \"And even if he caught the train, there was no avoiding a blow up with the boss, because the firm's errand boy would've waited for the five o'clock train and reported the news of his absence long ago.\", \"He was the boss's minion, without backbone or intelligence.\", 'Well then, what if he reported in sick?', \"But that would be extremely embarrassing and suspicious, because during his five years' service Gregor hadn't been sick even once.\", \"The boss would certainly come with the doctor from the health insurance company and would reproach his parents for their lazy son and cut short all objections with the insurance doctor's comments; for him everyone was completely healthy but really lazy about work.\", 'And besides, would the doctor in this case be totally wrong?', 'Apart from a really excessive drowsiness after the long sleep, Gregor in fact felt quite well and even had a really strong appetite.']\n",
```

Comment:
I wonder we can store them in small text files and load them from there instead of putting it directly

Reply:
or you can use black formating to make this inputs more readable